MAUI/ImageEditor/text.md

```diff
@@ -88,6 +89,38 @@ Customize the appearance of text annotation using the [`ImageEditorTextSettings`
 
 ![Text annotation customization in .NET Maui ImageEditor](images/text/imageeditor-text-customization.png)
 
+## Customize the background color of text annotations
+
+The background color of the text annotations can be customized using the `Background` property.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid RowDefinitions="0.9*, 0.1*">
+        <imageEditor:SfImageEditor x:Name="imageEditor"
```

Comment:
Mention Text color opacity code for programmatic annotation added

Reply:
Mentioned